src/components/ApplicationLayout/AppNavigation/AppNavigation.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import type { PropsWithSpread } from "types";
+import classNames from "classnames";
+import type { HTMLProps, PropsWithChildren } from "react";
+
+type Props = PropsWithSpread<
+  {
+    collapsed?: boolean;
+    pinned?: boolean;
+  } & PropsWithChildren,
+  HTMLProps<HTMLDivElement>
+>;
+
+const AppNavigation = ({
+  children,
+  className,
+  collapsed,
+  pinned,
+  ...props
+}: Props) => {
+  return (
+    <header
```

Comment:
Should this use the `nav` tag instead of `header`?

Reply:
The way this works in the [app layout examples](https://vanillaframework.io/docs/layouts/application) is that the `l-navigation` is a `<header>` and the links get wrapped in <nav>, which I've done inside [`SideNavigation`](https://github.com/canonical/react-components/pull/1031/files#diff-2342db5e0771288447f2934640857a99f923ff44cb2de93d573b3dca3eb0ce83R73).